src/main/java/hudson/model/ExternalJob.java

```diff
@@ -122,6 +122,39 @@ public String getDisplayName() {
             return Messages.ExternalJob_displayName();
         }
 
+        /**
+         * Needed if it wants External Monitor Jobs are categorized in Jenkins 2.x.
+         *
+         * TODO: Override when the baseline is upgraded to 2.x
+         *
+         * @return A string it represents a ItemCategory identifier.
+         */
+        public String getCategoryId() {
+            return "standaloneprojects";
```

Comment:
@amuniz @oleg-nenashev 

When this plugin upgrades his baseline, he will able to use something like that: `StandaloneProjectsCategory.ID`. By now, if we want that these plugins use ItemCategorization feature (provided by Jenkins 2.0) and additionally are compatible with Jenkins 1.x, this was the best option.